src/components/Individual_Article/Individual_Article/Article.jsx

```diff
@@ -15,14 +17,26 @@ const Article = ({articleId}) => {
     }
     if (isLoading) return <p>Loading...</p>
 
+    const handleVote = (vote) => {
+        patchVote(articleId, vote).catch(()=> {
+            setVoteChange((currVoteChange) => {return currVoteChange - vote})
+            setErr('Something went wrong, please try again.');
+        })
+        setVoteChange((currVoteChange) => {return currVoteChange + vote})
+        setErr(null);
+    }
+
     return (
     <section id="articleBox">
         <h2 className="heading" id="articleHeading">{article.title}</h2>
         <p id="author">Written by: {article.author}</p>        
         <p id="dateCreated">{article.created_at}</p>
         <p id="articleBody">{article.body}</p>
+        <p id="articleVotes">{article.votes + voteChange} likes</p>
         <img id='articleImg' src={article.article_img_url}/>
-        <button className='likeButton'>Like this article</button>
+        {err ? <p className="errMessage">{err}</p> : null}
+        <button className='likeButton' disabled={voteChange === 1} onClick={() => {handleVote(1)}}>Like this article</button>
+        <button className='dislikeButton' disabled={voteChange === -1}onClick={() => {handleVote(-1)}}>Dislike this article</button>
```

Comment:
I really like how you're handling this, disabling the buttons is great to stop votes being spammed